sign_up/sign_up.js

```diff
@@ -0,0 +1,64 @@
+// from homepage
+// sign up btn click sign up html
+// sign in btn click sign in html
+
+window.onload = () => {
+  registerNewUser();
+};
+
+const formElem = document.querySelector("form");
+const dialog = document.getElementById("dialog");
+const dialogWarning = document.getElementById("dialogWarningContent");
+const okBtn = document.getElementById("ok-btn");
+
+function closeDialog() {
+  dialog.classList.remove("active");
+  overlay.classList.remove("active");
+}
+
+okBtn.addEventListener("click", closeDialog);
+
+formElem.addEventListener("submit", (event) => {
+  event.preventDefault();
+});
+
+const usersState = {
+  name: null,
+  surname: null,
+  email: null,
+  password: null,
+  confirmPwd: null,
```

Comment:
confirm password is not a data field, it is only used to make sure the user didn't misspell their password.